### DISCUSSION TITLE
FR: Add Category to Devices panel

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#20464] [Dashboard]
Asking because not only physical devices are represented in that list `/config/devices/dashboard`, but also eg HACS integrations.

Aware we could add a label, but this would be a perfect spot to add a Category, and visually list the devices accordingly, without having the need to refer to them in the backend (for which purpose the labels where introduced)

**Comment by arturpragacz** [comment]
Labels don't fully replace categories, because there is no possibility to group by label, only by category.

So I fully support this proposal, it would be a very useful addition.

For that matter, I'd like to have categories also on the entities page (`/config/entities`).
